boto/connection.py

```diff
@@ -1112,6 +1115,7 @@ def make_request(self, action, params=None, path='/', verb='GET'):
             http_request.params['Action'] = action
         if self.APIVersion:
             http_request.params['Version'] = self.APIVersion
+        statsd.increment('boto.request', tags=["action:%s" % action or 'unknown', "host:"])
```

Comment:
I'm guessing `host:` is set to manually be empty due to overload - but would it be helpful in tracking down a problematic host making too many calls?